### DISCUSSION TITLE
Which Zigbee revert firmware should I use?

### DISCUSSION THREAD

**Discussion by Timmiej93** [pvvx/ATC_MiThermometer#417] [Q&A] [answered]
In [ATC_MiThermometer/zigbee_ota](https://github.com/pvvx/ATC_MiThermometer/tree/master/zigbee_ota) I see quite a few firmwares for the other devices (MHO-C401, CGG1M, etc.), but I don't see one for the LYWSD03MMC. 

Which firmware should I use to revert from Zigbee to BT?

**Comment by pvvx** [comment, marked as answer]
https://github.com/pvvx/ZigbeeTLc#zigbee-ota

1141-020a-99993001-ATC_v46.zigbee
1141-0203-99993001-ATC_v46.zigbee
